telco-core/configuration/reference-crs/required/networking/Network.yaml

```diff
@@ -20,6 +20,8 @@ spec:
   #  namespace: app-ns-1
   #  rawCNIConfig: '{ "cniVersion": "0.4.0", "name": "add-net-2", "plugins": [ {"type": "macvlan", "master": "bond1", "mode": "private" },{ "type": "tuning", "name": "tuning-arp" }] }'
   #  type: Raw
-
+    
+  {{ if hasKey .spec "useMultiNetworkPolicy" }}
   # Enable to use MultiNetworkPolicy CRs
-  useMultiNetworkPolicy: true
+  useMultiNetworkPolicy: {{ .spec.useMultiNetworkPolicy }}
+  {{ end }}
```

Comment:
This should be updated under the kube-compare set rather than the reference-crs set...